Added sample scripts for request module blog

kushal-likhi opened the pull request:
Added sample scripts for `request module` blog.

Commit:
Added sample scripts for request module blog

zwigby commented:
I might have missed this on the last set but let's make sure we use 2 spaces for indentation.

zwigby commented:
Let's remove the initial PUT, POST, DELETE, PATCH examples because they are only going to return errors.

zwigby commented on request/customConfigurations.js:
```diff
@@ -0,0 +1,17 @@
+}, function(error, response, body){
+    if(error) {
+        console.log(error);
+    } else console.log(response.statusCode, body);
```
Also use { } with if or else. Makes it too easy to break when not using them. This goes for all if/else statements below.

Commit:
Added editorsconfig file

Commit:
refactored code samples with new indentations and if-else blocks.

kushal-likhi commented:
Hi, updated the indentation for all files, also added `.editorconfig` for ensuring 2 spaces via IDE reformats. Also updated the if-else blocks and ensured statements are in blocks, Also updated the blog document with the same.
Removed the PUT/POST/DEL/PATCH code.

zwigby commented:
Looks like there are conflicts, can you make sure those are resolved. my guess is pull from branch and then submit pull.

Commit:
Resolved merge conflicts.

kushal-likhi commented:
Hey! Sorry for that, resolved conflicts and pushed!
